### DISCUSSION TITLE
CoX scouter plugin

### DISCUSSION THREAD

**Discussion by thefriendlydish** [runelite/runelite#17466] [Q&A]
Hello, I've been having issues with the CoX scout plugins and am unable to use the !layout or have the Layout in the little display box in the corner. I have tried reinstalling runelite and turning plugins on and off. Has anyone ran into this problem before or have a fix?

**Comment by edafs** [comment]
I actually ran into this issue not too long ago. It turns out I had checked the "Large Raids" option. 
I turned it off, and it started working for me again, not sure if this will resolve your issue too.

![image](https://github.com/runelite/runelite/assets/10689492/f4f23880-79e5-4be7-8e61-fa5e71d2d699)
